chunks/lang-java/99-gradle.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Extract the description field from the JSON string
+description=$(echo "$GITPOD_WORKSPACE_CLASS_INFO" | jq -r '.description')
+
+# Extract the number of vCPUs from the description field
+# Assumes the format "8 vCPU, 16GB memory, 50GB disk"
+vcpus=$(echo "$description" | grep -oP '\d+ vCPU' | awk '{print $1}')
```

Comment:
Suggestion: You can also use `nproc`

```suggestion
vcpus=$(nproc)
```

Reply:
I've updated the script to use `gp top --json` as discussed internally.